Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Detect_Service.yml

```diff
@@ -474,7 +493,7 @@ tasks:
     view: |-
       {
         "position": {
-          "x": 1360,
+          "x": 1250,
           "y": 430
         }
       }
```

Comment:
For Line 503, should we add "Skip this branch if this automation/playbook is unavailable"?